mvp-spec/partials/openrtb-request-with-transmissions.json

```diff
@@ -0,0 +1,88 @@
+{
+    "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+    "at": 1, 
+    "cur": [ "USD" ],
+    "imp": [
+        {
+            "id": "1",
+            "bidfloor": 0.03,
+            "banner": {
+                "h": 250,
+                "w": 300,
+                "pos": 0
+            },
+            "ext": {
+                "prebid_sso": {
+                    "version": 0,
+                    "seed": {
+                        "version": 0,
+                        "transaction_id": 1234567,
+                        "publisher": "publisher.com",
+                        "source": {
+                            "domain": "publisher0.com",
+                            "timestamp": 1639589531,
+                            "signature": "12345_signature"
+                        }
+                    },
+                    "parents": [],
+                }
+            }
+        }
+    ],
+    "site": {
+        "id": "102855",
+        "cat": [ "IAB3-1" ],
+        "domain": "www.publisher.com",
+        "page": "http://www.publisher.com/1234.html ",
+        "publisher": {
+            "id": "8953",
+            "name": "publisher.com",
+            "cat": ["IAB3-1"],
+            "domain": "publisher.com"
+        }
+    },
+    "device": {
+        "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/537.13 (KHTML, like Gecko) Version/5.1.7 Safari/534.57.2",
+        "ip": "123.145.167.10"
+    },
+    "user": {
+        "id": "55816b39711f9b5acf3b90e313ed29e51665623f",
+         "ext":
+         {
+            "eids": 
+            [
+                {
+                    "id": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                    "atype": 1,
+                    "ext": 
+                    {
+                        "prebid_sso": 
+                        {
+                            "version": 0,
+                            "type": "prebid_id",
+                            "source": 
+                            {
+                                "domain": "operotor0.com",
+                                "timestamp": 1639589531,
+                                "signature": "12345_signature"
+                            }
+                        }
+                    }
+                }
+            ],
```

Comment:
We use the existing `eids` now for sharing the Pseudonymous-Identifiers. We remove the `value` field of the existing `identifier` structure shared across other API to put in the `id` field described in the `eids`.

As a memo, usually, the Pseudonymous-Identifier object looks like this:
```json
{
   "version": 0,
    "type": "prebid_id",
     "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
      "source": {
          "domain": "operotor0.com",
          "timestamp": 1639589531,
           "signature": "12345_signature"
       }
 }
```

I have the feeling that this proposal is a bit odd. Please challenge it. Maybe we can find another way.